akka-persistence/src/main/resources/reference.conf

```diff
@@ -41,6 +41,12 @@ akka.persistence {
     no-snapshot-store {
       class = "akka.persistence.snapshot.NoSnapshotStore"
     }
+    recovery {
+      # There is more time in between individual events gotten from the journal
+      # during replay than this the replay will be failed and behave which causes
+      # the actor to stop (after onRecoveryFailure has been invoked)
+      event-timeout = 30s
+    }
```

Comment:
perhaps move it into the `journal-plugin-fallback` section, so that it's possible to use different values for different journal plugins, see how it's done for `max-message-batch-size`